webclient/src/forms/LoginForm/LoginForm.tsx

```diff
@@ -1,73 +1,73 @@
-// eslint-disable-next-line
```

Comment:
Not sure why your PR has these non-changes, but try to avoid adding them in the future.